dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/data/KnownJobData.kt

```diff
@@ -37,7 +38,7 @@ class KnownJobData constructor(
                 } else if (jobType == "PUT") {
                     Ds3ClientHelpers.wrap(client, RETRY_TIME).recoverWriteJob(jobId)
                 } else {
-                    throw NullPointerException("Got here")
+                    throw ConfigurationException("Was expecting GET or PUT, but got" + jobType)
```

Comment:
any reason your not using a string template? Also, for readability you need a space between the `...but got` and the job type

Reply:
I'm in Kotlin! I can do that!